src/csvcubed/writers/skoscodelistwriter.py

```diff
@@ -190,11 +196,17 @@ def _get_catalog_metadata(self, scheme_uri: str) -> ConceptSchemeInCatalog:
     def _get_code_list_data(self) -> pd.DataFrame:
         data_frame = pd.DataFrame(
             {
+                "Uri Identifier": [
+                    c.uri_safe_identifier for c in self.new_code_list.concepts
+                ],
                 "Label": [c.label for c in self.new_code_list.concepts],
                 "Notation": [c.code for c in self.new_code_list.concepts],
-                "Parent Notation": [c.parent_code for c in self.new_code_list.concepts],
+                "Parent Uri Identifier": [
+                    uri_safe(c.parent_code) if c.parent_code else None
```

Comment:
Instead of doing this (which I did suggest), I think we need to find the parent concept by its code and look up its `uri_safe_identifier`. This is because a user can override the parent's `uri_safe_identifier` and make it something which isn't just the `uri_safe(notation)` that it generally is.

Reply:
@robons I've updated accordingly.